### ISSUE TITLE
Chart Color

### ISSUE THREAD

**Issue by ChrisSantiago82** [PHPOffice/PhpSpreadsheet#1388]
This is:

```
- [X] a bug report
- [X ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
If I set the colors for a Linechart, I'm expecting the lines to have those colors.

### What is the current behavior?
They don't have those colors

### What are the steps to reproduce?
Take the sample file 33_Chart_create_line.php, and add a color.

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

### Which versions of PhpSpreadsheet and PHP are affected?
the latest.

My question is if this is only not supported? Or if it's a bug? Or am I doing something wrong?

**Comment by ChrisSantiago82** [comment]
Yes, it's better to close it than to fix bugs...

**Comment by oleibman** [comment]
Addressed by PR #2898 in July 2022.
